c/frame/_repr_html_.cc

```diff
@@ -178,18 +180,18 @@ class HtmlWidget {
 
 
     void render_table_footer() {
-      html << "  <div class='footer'>";
+      html << "  <div class='footer'>\n";
       render_frame_dimensions();
```

Comment:
What is the reason you use `\n`s here, and don't use them in `render_data_row()`? I guess they're only required for HTML code to look nice, when opened through "View Source" or something like that?

Reply:
Yes, from HTML standpoint the newlines don't matter. However, we put that html into the docs (see  docs/using-datatable.rst for example), and we'd like to have at least some amount of readability there.